source/css/tailwind.css

```diff
@@ -17,19 +17,19 @@
 }
 
 h5 {
-  @apply text-xl;
+  @apply text-xl; 
 }
 
 h6 {
-  @apply text-lg;
+  @apply text-lg; 
 }
 
 @tailwind components;
 
 @tailwind utilities;
 
 .feature {
-    height: 100vh;
+    height: 100vh; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/manual/Driminary/thomas-anderson.net/pullRequest?prid=5875174)